notebooks/2022-04-03potential_bot.ipynb

```diff
@@ -0,0 +1,162 @@
+    "                    'downloaded', 'services', 'development', 'drug', 'top', 'reviews',\n",
+    "                    'review', 'treatment', 'deliver', 'hire', 'hiring', 'supply', 'supplier', \n",
+    "                    'commercial', 'discount', 'account', 'websites', 'app', 'products', \n",
+    "                    'guests', 'deals'\n",
+    "                    ]\n",
+    "    count = len(flaggedWords)\n",
+    "    counter = 0\n",
+    "    while counter <= count:\n",
+    "        if word.lower() == flaggedWords[counter]:\n",
+    "            return True\n",
+    "            break\n",
+    "        else: \n",
+    "            counter = counter + 1\n",
+    "        if counter == count:\n",
+    "            return False\n",
+    "\n",
+    "\n",
+    "def unhelpful_word(word):\n",
+    "    unhelpfulWords = [\n",
+    "                      'col','plugin','rows','state','children','content','text',\n",
+    "                      'type','level','children','p','h','ttachment','ile','layout',\n",
+    "                      'child','image','src','width','alt','href'\n",
+    "                     ]\n",
+    " \n",
+    "    count = len(unhelpfulWords)\n",
+    "    counter = 0\n",
+    "    while counter <= count:\n",
+    "        if word.lower() == unhelpfulWords[counter]:\n",
+    "            return True\n",
+    "            break\n",
+    "        else: \n",
+    "            counter = counter + 1\n",
+    "        if counter == count:\n",
+    "            return False\n",
+    "        \n",
+    "        \n",
+    "def readable(splitString):\n",
+    "    resultData = ''\n",
+    "    for word in splitString:\n",
+    "        if word.isalpha():\n",
+    "            if unhelpful_word(word):\n",
+    "                pass\n",
+    "            else:\n",
+    "                resultData = resultData + ' ' + word\n",
+    "    return(resultData)\n",
+    "\n",
+    "\n",
+    "def description_length(description):\n",
+    "    descriptionLen = len(description)\n",
+    "    return (descriptionLen)\n",
+    "\n",
+    "def query(sql):\n",
+    "    c = db.cursor()\n",
+    "    c.execute(sql)\n",
+    "    return c.fetchall()\n",
+    "\n",
+    "potentialBots = np.empty((0,2))\n",
+    "\n",
+    "database = query('''SELECT id, description FROM user WHERE description != \"NULL\"''' )\n",
+    "\n",
+    "for user in database:\n",
+    "    flaggedWordCount = 0\n",
+    "    joinedString = \"\".join(user[1])\n",
+    "    splitString = re.split('\"|/.| ', joinedString)\n",
+    "    descLen = description_length(splitString)\n",
+    "    \n",
+    "    for word in splitString:\n",
+    "        if flagged_word(word):\n",
+    "            flaggedWordCount = flaggedWordCount + 1\n",
+    "    if flaggedWordCount >= 1:\n",
+    "        potentialBots = np.vstack([potentialBots, [user[0], readable(splitString)]])\n",
+    "    elif descLen > 1500:\n",
```

Comment:
could we find a descriptive name to this [magical number](https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad)? Something like MAX_DESCRIPTION_SIZE or something better ?